home/forms/search.py

```diff
@@ -1,8 +1,8 @@
 from copy import deepcopy
 from urllib.parse import urlencode
 
+from data_platform_catalogue.entities import FindMoJDataEntityType
```

Comment:
I am not sure we need to prefix with `FindMoJData`. that is the app name, the lib should be reusable independent of what the app is called.

if an app client has a specific data model, then it should implement its own entities map and convert from `EntityTypes` to it.